jekyll/_cci2/webhooks.md

```diff
@@ -61,19 +61,70 @@ Webhooks are set up on a per-project basis. To get started:
 ## Payload signature
 {: #payload-signature}
 
-You can provide an optional signing secret when setting up a webhook. If that
-secret is set, each outgoing HTTP request will contain a `circleci-signature`
-header, containing a comma-separated list of versioned signatures:
+You should validate incoming webhooks to verify that they are coming from
+CircleCI. To support this, when creating a webhook, you can optionally provide a
+secret token. Each outgoing HTTP request to your service will contain a
+`circleci-signature` header. This header will consist of a comma-separated list
+of versioned signatures.
 
 ```
 POST /uri HTTP/1.1
-Host: webhook-host
-circleci-signature: v1=signature1,v2=signature2,...
+Host: your-webhook-host
+circleci-signature: v1=4fcc06915b43d8a49aff193441e9e18654e6a27c2c428b02e8fcc41ccc2299f9,v2=...,v3=...
 ```
 
-Currently only the "v1" signature will be provided, and it is computed by
-running `HMAC-SHA256` on the request body, using the configured signing secret as
-the secret key.
+Currently, the latest (and only) signature version is v1. You should *only*
+check the latest signature type to prevent downgrade attacks.
+
+The v1 signature is the HMAC-SHA256 digest of the request body, using the
+configured signing secret as the secret key.
+
+Here are some example signatures for given request bodies:
+
+| Body                           | Secret Key       | Signature                                                          |
+| ------------------------------ | ---------------- | ------------------------------------------------------------------ |
+| `hello world`                  | `secret`         | `734cc62f32841568f45715aeb9f4d7891324e6d948e4c6c60c0621cdac48623a` |
+| `lalala`                       | `another-secret` | `daa220016c8f29a8b214fbfc3671aeec2145cfb1e6790184ffb38b6d0425fa00` |
+| `an-important-request-payload` | `hunter123`      | `9be2242094a9a8c00c64306f382a7f9d691de910b4a266f67bd314ef18ac49fa` |
+{: class="table table-striped"}
```

Comment:
Perhaps not an issue, but this seems to have rendered itself as a new row in the table of examples.

Reply:
This worried me too, but it seems to be consistent with all the other tables. I'll double check the prod docs after merging to be sure.